### PR TITLE
Added Main view

### PR DESCRIPTION
**Pull request by danielditer**

### CODE REVIEW

File: src/main/java/com/fundation/search/view/MainView.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+package com.fundation.search.view;
```
Comment (PanDeBatalla94) [inline src/main/java/com/fundation/search/view/MainView.java]: copyright missing

File: src/main/java/com/fundation/search/view/MainView.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        panelTabbedPane = new JPanel();
+        //======== this ========
+        Container contentPane = getContentPane();
+        contentPane.setLayout(new MigLayout(
+                "hidemode 3",
+                // columns
+                "[fill]" + "[fill]",
+                // rows
+                "[]" + "[]" + "[]"));
+        //======== menuBar ========
```
Comment (manu863018) [inline src/main/java/com/fundation/search/view/MainView.java]: Comments could be written this way 
/*
 * Here is a block comment.
 */

File: src/main/java/com/fundation/search/view/MainView.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import javax.swing.JPanel;
+
+import info.clearthought.layout.TableLayout;
+import net.miginfocom.swing.MigLayout;
+
+/**
+ * @version
+ * 27 Mar 2018  * @Daniel Caballero.
+ */
+public class MainView extends JFrame {
```
Comment (PanDeBatalla94) [inline src/main/java/com/fundation/search/view/MainView.java]: description class is missing

**Comment by JimmyRomero** [comment]
LGTM!

**Pull request merged**